theurgy.lic

```diff
@@ -479,22 +689,29 @@ class Theurgy
     Flags.add('theurgy-kertigen', 'The thick smell of ozone fills your nostrils', 'The sounds of Kertigen\'s forge still ring in your ears')
     Flags.add('research_done', '^Breakthrough!')
 
-    ######################## SCRIPT ########################
-
+    @actions = TheurgyActions.new(@settings, @merged_data)
+    
     pray
-    check_supplies
-    walk_to '741'
-    fput 'commune sense'
-    waitrt?
-
-    ritual_order = %i[tithe study_wall reflectory carve_bead find_seed enter_altar bathe meditate_bead pray_badge dance incense_wine recite_prayer clean_altar pray exit_altar invoke_parchment]
-    ritual_order.each do |ritual|
-      commune_check
-      send(ritual)
-    end
+    @actions.buy_supplies
+    @actions.perform_next_action while @actions.rituals_remain? and should_continue?
```

Comment:
We've been using `&&` (instead of `and`)

Reply:
Done.